src/components/HypothesisBuilder/components/EvidenceSection/index.js

```diff
@@ -33,14 +30,18 @@ export function EvidenceSection({
           Add evidence
         </button>
       </div>
-      <div className="border mt-2">
-        {evidence && evidence.length > 0 ? (
-          evidence.map((evidence) => (
-            <EvidenceItem key={evidence.id} evidence={evidence} />
-          ))
-        ) : (
-          <div className="py-2 px-4 text-sm text-gray-400">No evidence</div>
-        )}
+      <div className="mt-3.5">
```

Comment:
We can't remove the Evidencetem it is rendering the actual data that needs be present - we should be styling that data, not removing it

Reply:
<img width="745" alt="Screenshot 2022-02-04 at 10 11 11 AM" src="https://user-images.githubusercontent.com/32077311/152495000-5ccc7091-22de-474e-8238-d5233c16a63d.png">
<img width="776" alt="Screenshot 2022-02-04 at 10 15 58 AM" src="https://user-images.githubusercontent.com/32077311/152495003-c495a96c-f725-4164-a732-8abb99b96b5c.png">
About EvidenceItem, now if type not log or topology rendering data with style from design. But I noticed evidence data always have type log and because renders LogTable. Is it expected behavior?

Reply:
Yes, if evidence type is log it will render log table or if topology it will render a card